public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I don't think this file is needed to fulfil the brief, and this app isn't a PWA. Can it be removed?

Reply:
Done! I learned what PWAs mean as well.